Trakt/ScheduledTasks/SyncFromTraktTask.cs

```diff
@@ -500,6 +462,53 @@ private async Task SyncTraktDataForUser(Jellyfin.Data.Entities.User user, double
         while (previousCount != 0);
     }
 
+    private bool UpdateEpisodeData(Jellyfin.Data.Entities.User user, Episode episode, UserItemData userData, DateTime? tLastPlayed, int plays)
```

Comment:
You can import `Jellyfin.Data.Entities` if there are no conflicts and use `User` directly.

Reply:
ok